docs-site-feature-tests/index.md

```diff
@@ -74,3 +74,5 @@ You should see a list of 5 HTTP Request templates
 You should see the AI Glossary below
 
 --8<-- "_glossary/ai-glossary.md"
+
+Link to [workflows](/glossary/#workflow-n8n){ data-preview } glossary definition with preview.
```

Comment:
This will not actually point to anything until I merge a change to change the glossary to use headings, but I wanted to set this up ahead of time to be able to test.